### PR TITLE
Add secure keys

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,20 @@
 sudo: false
 language: node_js
 node_js:
-  - "5.5"
+- '5.5'
```
Comment (mgol) [inline .travis.yml]: I'd change that to '6'. First of all, Node 5 is EOL'd in about a month, secondly, we shouldn't pin to a minor as only the latest n.x.y for each n is supported by upstream.

Comment (leobalter) [inline .travis.yml]: that was the travis cli chaning the file. I'll update this to Node 6 in a new commit.

**Comment by leobalter** [comment]
Node is updated

**Comment by leobalter** [comment]
that's still unavailable for PRs, but better to have the _explicit_ configuration as in this PR

**Comment by mgol** [comment]
LGTM. The variables need to be then removed from https://travis-ci.org/jquery/jquery-color/settings. We also need to skip BrowserStack testing on PRs, logic akin to https://github.com/jquery/sizzle/blob/ed285e39a210bc7a3bf3af0f1145832076146253/Gruntfile.js#L259-L269 is needed.

Should I prepare a PR after this one lands>=?

**Comment by raphamorim** [comment]
LGTM too,
Yes @mgol, I think it's even better put this [logic](https://github.com/jquery/sizzle/blob/ed285e39a210bc7a3bf3af0f1145832076146253/Gruntfile.js#L259-L269) in other PR :octocat: :D

**Comment by leobalter** [comment]
this is now rebased with the changes from #103, this might be good to run browserstack even on PRs as we do on QUnit.

**Comment by leobalter** [comment]
this is using the same thing from QUnit, I don't know what is really happening right now

**Comment by mgol** [comment]
Note that BrowserStack has just notified us they are implementing support for the [Travis JWT add-on](https://docs.travis-ci.com/user/jwt) so this setup will require refactoring soon to make BrowserStack available for PRs without disclosing the key.

**Comment by leobalter** [comment]
Yes, but I'm planning to fix this here and on QUnit when it's available, in the meanwhile, this wouldn't case much harm. I'd rather let this shipped and working

**Comment by raphamorim** [comment]
💐  lol

**Pull request merged**